server/hotrod/src/main/java/org/infinispan/server/hotrod/HotRodOperation.java

```diff
@@ -73,6 +73,17 @@
    COUNTER_REMOVE(0x5E, 0x5F, EnumSet.of(OpReqs.REQUIRES_AUTH), DecoderRequirements.HEADER_CUSTOM),
    COUNTER_GET_NAMES(0x64, 0x65, EnumSet.of(OpReqs.REQUIRES_AUTH), DecoderRequirements.HEADER_CUSTOM),
 
+   // Multimap operations
+   GET_MULTIMAP(0x67, 0x68, EnumSet.of(OpReqs.REQUIRES_KEY, OpReqs.REQUIRES_AUTH, OpReqs.CAN_SKIP_CACHE_LOAD, OpReqs.IS_MULTIMAP), DecoderRequirements.KEY),
```

Comment:
We were using the same numbers I had conflicts on that !

Reply:
In any case, at one point this should evolve because we are getting rig of bytes and response codes from server to client are handled as shorts

https://issues.jboss.org/browse/ISPN-8590